Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedBackingStore.cpp

```diff
@@ -93,8 +93,34 @@ void CoordinatedBackingStore::paintToTextureMapper(TextureMapper& textureMapper,
     FloatRect layerRect = { { }, m_size };
     TransformationMatrix adjustedTransform = transform * TransformationMatrix::rectToRect(layerRect, targetRect);
 
+#if ENABLE(DAMAGE_TRACKING)
+    const auto damageRects = textureMapper.damage().rects();
```

Comment:
```suggestion
    const auto& damageRects = textureMapper.damage().rects();
```

so we avoid an unnecessary copy